### ISSUE TITLE
CET crashes the game in the heist mission

### ISSUE THREAD

**Issue by H4vC** [yamashi/CyberEngineTweaks#710]
This is essentially the same as #653 with all options off.
Crash happens when V enters Delamain and talks to Jackie.
I can sadly only confirm this happens on AMD ryzen (2700x 3950x and a TR) Nvidia Gpu (3090, 2080) as these are the machines I can easily access.

**Comment by WSSDude420** [comment]
Fixed by https://github.com/yamashi/CyberEngineTweaks/commit/22017dd6717b9edb11f32fde2e0e82d2588728a4

**Comment by ArsenicTouch** [comment]
Provide a link to your save so it can be tested.

**Comment by ArsenicTouch** [comment]
Also, there;'s a known crash issue during the heist with the delmain, if you have the boundary teleport and the minimap flicker turned on under the options menu for CET.

**Comment by Tiffys** [comment]
I experienced something similar with the minimap flicker and disable boundary teleport unchecked. Though what I did have checked was Remove Dead Bindings, Enable Debug Menu, Skip Start Menu, Suppress Intro Movies - some of those being in the dev tab. 

Unchecking those I was able to get past the conversation with Jackie outside Afterlife and complete The Heist without further issues. This character was created in 1.4 with an upgrade to 1.6 after creation - not sure if that's relevant. As for my hardware it's a Ryzen 9 5900X, and a 3080. Game's on an 8TB M.2 NVMe.
